WangscapeTest/TestCalculatorMax.cpp

```diff
@@ -0,0 +1,81 @@
+#include <gtest/gtest.h>
+#include <random>
+#include <numeric>
+#include <algorithm>
+
+#include <tilegen/alpha/CalculatorMax.h>
+
+class TestCalculatorMax : public ::testing::Test
+{
+public:
+    tilegen::alpha::CalculatorMax cm;
+    std::mt19937 rng;
+    TestCalculatorMax()
+    {
+        std::random_device rd;
+        rng.seed(rd());
+    }
+    int countValue(const tilegen::alpha::CalculatorBase::Alphas& alphas, int value)
+    {
+        return std::accumulate(alphas.cbegin(), alphas.cend(), 0,
+                               [&value](int acc, sf::Uint8 x)
+        { return (x == value) ? acc + 1 : acc; });
```

Comment:
There are some `std::accumulate` calls already in `master` which could be replaced with `count_if`, and I think the `CartesianPower` tests have a for loop that could be replaced by `count`. I think that's a matter for another PR, though.